rom/dev/src/flow/cold_reset/fmc_alias.rs

```diff
@@ -252,6 +252,6 @@ impl FmcAliasLayer {
             flags |= dice::FLAG_BIT_DEBUG;
         }
 
-        flags.to_be_bytes()
+        flags.reverse_bits().to_be_bytes()
```

Comment:
Why do the bits need to be reversed? The constants seem to match the spec.

Reply:
Our understanding for a DER encoded ASN1 bit string is that for the actual value (the flags) the most-significant-bit of the first byte transferred corresponds to bit 0 of the flags.

See this example: https://learn.microsoft.com/en-us/windows/win32/seccertenroll/about-bit-string

Reply:
Ah, you're right. Copying from our chat discussion

Looking at the ASN.1 spec: https://www.itu.int/ITU-T/studygroups/com17/languages/X.680-0207.pdf

Section 21.2
> The first bit in a bit string is called the leading bit. The final bit in a bit string is called the trailing bit.

Section 21.4
> The leading bit of the bit string is identified by the "number" zero, with succeeding bits having successive values.

Reply:
Oh nice this is the point of truth we needed. I was only finding references that implied this. Thanks for getting to the bottom of this!